src/components/molecules/GalleryGrid/GalleryGrid.jsx

```diff
@@ -0,0 +1,21 @@
+import gallery1 from "../../atoms/assets/images/gallery_img_1.png";
```

Comment:
use data from JSON to import images, don't import them manually one by one(if you will have 100 images it will be time consuming).

Reply:
I did it , but I have one question regarding it, because I'm a little but confused. 
For such approach I changed image placement to /public folder, but is it possible to get them form /src/assets/... folder? 
Tried to find solution, but change from src to public is only solution I can find and use.